.php_cs.dist

```diff
@@ -43,6 +43,7 @@ return PhpCsFixer\Config::create()
         'self_accessor' => false,
         'yoda_style' => null,
         'single_line_throw' => false,
+        'no_alias_language_construct_call' => false,
```

Comment:
This is a new [rule](no_alias_language_construct_call) `no_alias_language_construct_call` that replace all `die()` by `exit()`.

This can be discussed and maybe enabled in another PR. But enabling it means changing thousand of files so I'm not sure this is valuable.

Reply:
So I disable it for now.

Reply:
Ok for me.